share/functions/fish_add_path.fish

```diff
@@ -47,7 +47,14 @@ function fish_add_path --description "Add paths to the PATH"
         set -l p (builtin realpath -s -- $path 2>/dev/null)
 
         # Ignore non-existing paths
-        test -d "$p"; or continue
+        if not test -d "$p"
+            # path does not exist
+            if set -q _flag_verbose;
+                # print a message in verbose mode
+                echo (_ "Skipping non-existent path:" " $p")
```

Comment:
the `$p` should be *outside* of the translation, or it will attempt to translate your path.

In full it would probably be:

```fish
printf (_ "Skipping non-existent path: %s\n") "$p"
```

This allows translations to have the full context and put the path elsewhere if needed.